.github/workflows/build.yaml

```diff
@@ -30,3 +30,10 @@ jobs:
           fetch-depth: 1
       - name: Build
         run: make
+      - name: Run markdownlint with auto-fix
+        uses: DavidAnson/markdownlint-cli2-action@05f32210e84442804257b2a6f20b273450ec8265 #v19
```

Comment:
I hope this works; the examples I have seen all have a space after the `#`. Unfortunately it isn't documented how exactly dependabot is updating comments.